src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -383,10 +380,13 @@ public void Dispose()
 
                 _isDisposed = true;
 
-                _abortCts.Dispose();
                 _abortRegistration.Dispose();
 
                 _fetchResponse?.Dispose();
+                if (_abortController != null && !_abortController.IsDisposed)
```

Comment:
I think this is already covered by by `abortRegistration` which invokes `abort` on `abortController`.

Reply:
the registration would not call it unless the cancelationToken was fired. This covers then case when it was disposed but the cancelationToken did not trigger.